examples/3d_kymograph.py

```diff
@@ -0,0 +1,148 @@
+
+    _tmp = np.asarray(list(idr_plane_iterator))
+    _tmp = _tmp.reshape((nz, nc, nt, ny, nx))
+    # the following line reorders the axes (no summing, despite the name)
+    return np.einsum("jmikl", _tmp)
+
+
+description = """
+3D-Kymographs in Napari
+=======================
+
+About
+=====
+This example demonstrates that the volume rendering capabilities of napari
+can also be used to render 2d timelapse acquisitions as kymographs. 
+Kymographs, also called space-time images, are a powerful tool to visualize
+the dynamics of processes.  
+The most common way to visualize kymographs is to pick a single line through
+a 2D image and visualize the time domain along a second axes. 
+Napari is not limited to 2D visualization an by harnessing its volume
+volume rendering capabilities, we can create a 3D kymograph, 
+a powerful visualization that provides an overview of the complete 
+spatial and temporal data from a single view.
+
+Using napari's grid mode we can juxtapose multiple such 3D kymographs to
+highlight the differences in cell dynamics under different siRNA treatments. 
+
+The selected samples are from the Mitocheck screen and demonstrate siRNA 
+knockdowns of several genes. 
+The date is timelapse fluorescence microscopy of HeLa cells, with GFP-
+tagged histone revealing the chromosomes.
+
+In the juxtaposed kymographs the reduced branching for the mitotitic 
+phenotypes caused by INCENP, AURKB and KIF11 knockdown compared to 
+TMPRSS11A knockdown is immediately obvious.
+
+Data Source
+===========
+The samples to demonstrate this is downloaded from IDR:
+https://idr.openmicroscopy.org/webclient/?show=screen-1302
+
+Reference
+=========
+The data comes from the Mitocheck screen:
+
+Phenotypic profiling of the human genome by time-lapse microscopy reveals cell
+division genes. 
+
+Neumann B, Walter T, Hériché JK, Bulkescher J, Erfle H, Conrad C, Rogers P,
+Poser I, Held M, Liebel U, Cetin C, Sieckmann F, Pau G, Kabbe R, Wünsche A,
+Satagopam V, Schmitz MH, Chapuis C, Gerlich DW, Schneider R, Eils R, Huber W,
+Peters JM, Hyman AA, Durbin R, Pepperkok R, Ellenberg J.
+Nature. 2010 Apr 1;464(7289):721-7. 
+doi: 10.1038/nature08869. 
+
+Acknowledgements
+================
+Beate Neumann (EMBL) for helpful advice on mitotic phenotypes.
+
+"""
+
+print(description)
+
+samples = (
+    {"IDRid": 2864587, "description": "AURKB knockdown", "vol": None},
+    {"IDRid": 2862565, "description": "KIF11 knockdown", "vol": None},
+    {"IDRid": 2867896, "description": "INCENP knockdown", "vol": None},
+    {"IDRid": 1486532, "description": "TMPRSS11A knockdown", "vol": None},
+)
+
+print("------------------------------------")
+print("Downloading sample datasets from IDR")
+print("this may take some time")
+print("------------------------------------")
+for s in samples:
+    print(f"Downloading sample {s['IDRid']}.")
+    print(f"Description: {s['description']}")
+    s["vol"] = np.squeeze(IDR_fetch_image(s["IDRid"]))
```

Comment:
Does this script re-download the data every time it is run?
It would be more convenient if we check whether the data has already been downloaded before, and if not then download it.

Reply:
@VolkerH and I had a conversation about this offline. My opinion was that it was a bit obnoxious to download something to a not-well-defined place (e.g. current directory). The long term solution is to depend on pooch or similar, but I figure for our examples and for now (pre-tutorials/gallery), re-downloading is ok.

Reply:
yes, it does.
This current PR is trimmed down version of [a larger example](https://github.com/VolkerH/napari/blob/regionprops/examples/display_regionprops.py) that includes caching of the downloaded data in a Zarr store on disk.

However, I felt that all of the caching and segmentation code in the larger example obscures the view onto the relevant parts. Even in the current example, a large fraction of the code just deals with downloading the sample data and not with showcasing napari's functionality. I put the larger example on ice for now, but it may be something to revisit for a tutorial. I just don't think the main napari repo would be the right place.

Reply:
Those are good points. I guess we should just make it super clear to the user about how it works & our long term plan can be some kind of pooch solution.